### PR TITLE
[ADP-3332] Update E2E instructions

### PR DESCRIPTION
**Pull request by paolino**
- [x] Add a fetch-preprod-snapshot script that target new URL for snaphsots
- [x] Use it in linux , macos and docker e2e
- [x] Fix windows snapshot downloading instructions
- [x] Remove ad-hoc and broken E2E node config files and link the toplevel ones instead

### CODE REVIEW

File: scripts/gh/fetch-preprod-snapshot.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -e -u -o pipefail
```
Comment (HeinrichApfelmus) [inline scripts/gh/fetch-preprod-snapshot.sh]:
```suggestion
set -euo pipefail
```

? 🤷

**Review by HeinrichApfelmus — APPROVED** [review]
Thank you! 😊

File: scripts/gh/fetch-preprod-snapshot.sh (hunk 1 above)
Comment (paweljakubas) [reply scripts/gh/fetch-preprod-snapshot.sh]: :smile:

**Review by paweljakubas — APPROVED** [review]
thanks! lgtm!

**Pull request merged**